### ISSUE TITLE
Update tutorials for DIALS 2.0

### ISSUE THREAD

**Issue by dagewa** [dials/dials#928]
These tutorials need testing and updating for DIALS 2.0
- [ ] Processing in Detail with DUI
- [x] Processing in Detail
- [x] Scaling of Beta-Lactamase dataset
- [x] Multi-crystal symmetry analysis and scaling with DIALS
- [x] Small Molecule Tutorial
- [x] DPF3 Part 1: Correcting poor initial geometry
- [x] DPF3 Part 2: A question of centring
- [x] Multi-lattice Tutorial
- [ ] ~~DIALS for Electron Diffraction~~ currently broken in 2.0
- [ ] Developer Tutorial
- [x] Processing in Detail (Thaumatin)
- [x] Using DIALS at Diamond Light Source
- [ ] Refining multi-tile detector metrology with DIALS ?
- [x] Multi-crystal analysis with DIALS and BLEND ?
- [ ] DIALS cheat sheet

**Comment by jbeilstenedmands** [comment]
Note that there is also a tutorial 'Refining multi-tile detector metrology with DIALS' that is not in the main index but is linked to in 'Using DIALS at Diamond Light Source'. It has an out of date warning at the top and no link to the actual datafile (it's a thaumatin dataset collected at diamond but looks like it's different to the standard thaumatin dataset?). However the content looks interesting so perhaps worth fixing and moving to advanced/developer section in the main index?

**Comment by jbeilstenedmands** [comment]
Two more added to list (and moved into the tutorials index in `dials-tutorial-improvements` branch), which are currently only referenced from 'Using DIALS at Diamond'.
These tutorials are both fairly involved, interesting and out-of-date, so some consensus requested on plans for DIALS2.0 docs.

**Comment by Anthchirp** [comment]
@noemifrisina found that the DPF tutorial is not generated/updated like the betalactamase/thaumatin tutorials are. The effect is that the DIALS output shown in the tutorial is 3 years old, which is not quite identical to the current output any more. Should we make this into a live-updated thing?

**Comment by jbeilstenedmands** [comment]
It would be nice to have this kind of thing for all of the tutorials, however does this create a high cost on the daily documentation building? Also worth noting that David has updated this tutorial now in the dials-tutorial-improvement branch. Perhaps it's more practical to check every _x_ months that the tutorials are up to date?

**Comment by graeme-winter** [comment]
Certainly for the less-frequently-followed paths ever so slightly out of date documentation can easily be forgiven, unless this is _also_ testing a useful code path?

**Comment by Anthchirp** [comment]
We build the documentation on (basically) every commit, but we run the tutorial commands only once a week (logs go into https://github.com/dials/dials.github.io/commits/master/tutorial_data), so this does not cost us that much.

**Comment by dagewa** [comment]
> Note that there is also a tutorial 'Refining multi-tile detector metrology with DIALS' that is not in the main index but is linked to in 'Using DIALS at Diamond Light Source'. It has an out of date warning at the top and no link to the actual datafile (it's a thaumatin dataset collected at diamond but looks like it's different to the standard thaumatin dataset?). However the content looks interesting so perhaps worth fixing and moving to advanced/developer section in the main index?

I think the main problem with this tutorial is that although integration results did look _slightly_ better after metrology refinement, this didn't translate into significant improvements downstream, for a lot of extra work by the user and slower processing (treating each image as 60 separate panels comes with a cost). It might be better to replace with a metrology refinement tutorial for a case where it actually matters. @phyy-nx has done most of the work there. It would be nice, for example, if the processing done for [this paper](https://doi.org/10.1107/S2059798318009191) could be turned into a tutorial on the website.

**Comment by dagewa** [comment]
> @noemifrisina found that the DPF tutorial is not generated/updated like the betalactamase/thaumatin tutorials are. The effect is that the DIALS output shown in the tutorial is 3 years old, which is not quite identical to the current output any more. Should we make this into a live-updated thing?

The DPF3 tutorial is slightly crippled now. It was supposed to be an exploration of a difficult dataset with various problems, requiring interventions. One of those problems (setting gain in spot finding to get rid of noise along module boundaries) has gone away with _new improved_ spotfinding. Then a large section in the second part was removed for reasons outlined in f91ff28a. This is a problem with writing tutorials for tricky cases where fine details of the software behaviour are relevant, as these fine details tend to change. At the moment I would keep it as is but resolve to review its relevance at some point again in the future.

**Comment by phyy-nx** [comment]
@dagewa, what about linking to the tutorial I made for the paper?
https://github.com/phyy-nx/dials_refinement_brewster2018

**Comment by dagewa** [comment]
The DIALS [cheat sheet](https://dials.github.io/documentation/programs/index.html) is also out of date. Adding to the list above.

**Comment by dagewa** [comment]
@phyy-nx right, I'd forgotten you had actually done that already!
